WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSitePreviewViewModel.kt

```diff
@@ -161,13 +165,15 @@ class NewSitePreviewViewModel @Inject constructor(
             IDLE, CREATE_SITE -> {
             } // do nothing
             SUCCESS -> {
+                tracker.trackPreviewLoading()
```

Comment:
Would it be better to call this in `startPreLoadingWebView` instead? Or maybe the reason you have it here is to make it consistent with the error state? I think either one works, but I feel having in `startPreLoadingWebView` would have been better because it's literally tracking that.

Reply:
Yep, I did it because I wanted to be consistent with the error state. However, after looking at it again, I guess it makes more sense to move it. 👍 

Fixed in [#6254ef3](https://github.com/wordpress-mobile/WordPress-Android/pull/8998/commits/6254ef3aa271abf7be30bb26b4133ce373fc6e34)